config/data/base_genesis.json

```diff
@@ -26,6 +26,10 @@
       "address": "0x3c44cdddb6a900fa2b585dd299e03d12fa4293bc",
       "balance": "10000000000000000000"
     },
+    {
```

Comment:
what is this?

Reply:
added a new account for test scripts to use without changing the disseminator nonce